src/pdn/doc/add_pdn_ring.md

```diff
@@ -0,0 +1,41 @@
+## add_pdn_ring
+
+### Synopsis
+```
+  % add_pdn_ring
+    [-name grid_name] \
+    -layer layer_name \
```

Comment:
I think you could do what you did in `add_pdn_connect` and define the two layers here, `-layers {}`.
That way it only needs to be called once

Reply:
Ok -  I will go with this. There is a potential for the width/spacing requirements to be different for the two layers, so the -width and -spacing arguments become -widths and -spacings, such that they can take a single value to be applied to both layers, or a list of two values to be applied to their respective layers.

Reply:
that sounds reasonable.